src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs

```diff
@@ -148,14 +150,35 @@ public override MetadataFileResult MovieMetadata(Movie movie, MovieFile movieFil
 
                 details.Add(new XElement("studio", movie.Studio));
 
-                if (image == null)
+                if (thumbnail == null)
                 {
                     details.Add(new XElement("thumb"));
```

Comment:
This causes there to be an empty thumb element, even if later poster thumb elements are created. Not sure this would effect anything or not, but here is whats created:

```
  <thumb />
  <thumb aspect="poster">http://image.tmdb.org/t/p/original/84Dhwz93vCin6T1PX6ctSvWEuNE.jpg</thumb>
  <fanart>
    <thumb>http://image.tmdb.org/t/p/original/qEu2EJBQUNFx5mSKOCItwZm74ZE.jpg</thumb>
  </fanart>
```

Reply:
Regarding poster thumbnails, I just tested the effect of the empty tag on poster thumb selection and Kodi only seems to look for thumb elements where the aspect="poster" attribute is explicitly declared when searching for poster thumbs. So the empty tag is ignored when looking for poster thumbs inside the .nfo file.

Regarding video thumbnails, the empty thumb element will still cause video thumbnails to be generated in Kodi, as it was the undesired result that led to #2837. Therefore, I did not remove the code, since it was already present before I added support for poster thumbs. Excluding the empty thumb tag would not influence Kodis automatic video thumbnail generation.